ci/run_tests.sh

```diff
@@ -69,13 +69,13 @@ run()
         if [ "$1" = "to_fail" ]; then
             exit_status=0
             RESULT={{$(ssh -o "StrictHostKeyChecking no" -p ${SSHPORT} root@${SSHHOST} -t \
-	                   "setenv LD_LIBRARY_PATH /root && ./$(basename ${name})")} && exit_status=1} || true
+	                   "export  LD_LIBRARY_PATH=/root:\$\{LD_LIBRARY_PATH}\ && ./$(basename ${name})")} && exit_status=1} || true
```

Comment:
Only one space needed after `export` *but* it would be preferable to ditch the `export` and do something like:

```
"LD_LIBRARY_PATH=/root:\$\{LD_LIBRARY_PATH} ./$(basename ${name})"
```

to implicitly scope the `LD_LIBRARY_PATH` to only the one command which needs it.